lib/components/List/GenericManagementList.tsx

```diff
@@ -129,6 +129,10 @@ export class GenericManagementList<T> extends React.PureComponent<GenericManagem
         }
     };
 
+    constructor(props: Readonly<GenericManagementListProps<T>>) { 
```

Comment:
This doesn't seem to be used, is this needed?

Reply:
There's a compile error with newer React library without this constructor.